common/samsung_signature.ksy

```diff
@@ -0,0 +1,42 @@
+meta:
+  id: samsung_signature
+  title: Samsung Crypto Signature for Android Firmware Images
```

Comment:
I don't see any crypto in this spec.

Reply:
Only part of the fields reversed. If you know more, please add. Anyway, you also use it https://github.com/kaitai-io/kaitai_struct_formats/pull/302/files#diff-41ce83ce4a8bc1c0b8eb954dc851ef7a877bc4c92b13391a407fe2ffcbab6d2eR33-R34